protos/mission/mission.proto

```diff
@@ -7,7 +7,9 @@ option java_outer_classname = "MissionProto";
 
 service MissionService {
     rpc UploadMission(UploadMissionRequest) returns(UploadMissionResponse) {}
+    rpc UploadMissionCancel(UploadMissionCancelRequest) returns(UploadMissionCancelResponse) {}
```

Comment:
Wouldn't it be clearer to call that `CancelMissionUpload`? Wondering :slightly_smiling_face:.

Reply:
Yup, agree. I've been just following the C++ SDK naming. So the discussion it falls in the same category as the cancel calibration thing. I can Do the updates here first and you guys change the SDK later?

Reply:
I think so. But we need @julianoes to agree on that, too :-).